extensions/sql-database-projects/src/test/testUtils.ts

```diff
@@ -88,28 +88,34 @@ export async function createDummyFileStructure(createList?: boolean, list?: stri
 	for (let dirCount = 1; dirCount <= 2; dirCount++) {
 		let dirName = path.join(testFolderPath, `folder${dirCount}`);
 		await fs.mkdir(dirName, { recursive: true });
+
 		if (createList) {
 			list?.push(dirName);
 		}
 
 		for (let fileCount = 1; fileCount <= 5; fileCount++) {
 			let fileName = path.join(dirName, `file${fileCount}.sql`);
-			await fs.open(fileName, 'w');
+			await touchFile(fileName);
 			if (createList) {
 				list?.push(fileName);
 			}
 		}
 	}
 
 	filePath = path.join(testFolderPath, 'file2.txt');
-	await fs.open(filePath, 'w');
+	await touchFile(filePath);
 	if (createList) {
 		list?.push(filePath);
 	}
 
 	return testFolderPath;
 }
 
+export async function touchFile(filePath: string) {
+	const handle = await fs.open(filePath, 'w');
```

Comment:
What is this meant to be testing? Using stat is the typical way of touching a file

Reply:
Yes, that would have been a better way to handle it. I will update that.